internal/model/tempname.go

```diff
@@ -25,7 +26,9 @@ func (t tempNamer) IsTemporary(name string) bool {
 }
 
 func (t tempNamer) TempName(name string) string {
+	hash := md5.New()
+	hash.Write([]byte(name))
 	tdir := filepath.Dir(name)
-	tname := fmt.Sprintf("%s.%s", t.prefix, filepath.Base(name))
+	tname := fmt.Sprintf("%s.%x", t.prefix, hash.Sum(nil))
```

Comment:
I'd suggest to keep at least a part of the filename such that users can see what's going on
```
       tbase := filepath.Base(name)
       if len(tbase) > 100 {
               hash := md5.New()
               hash.Write([]byte(name))
               tbase = tbase[0:100] + hash.Sum(nil)
       }
       tname := fmt.Sprintf("%s.%s", t.prefix, tbase)
```